setup.sh

```diff
@@ -59,7 +59,6 @@ python csm_web/manage.py createtestdata --yes
 
 # TODO put precommit hooks in git?
 echo "Installing pre-commit hook..."
-pip3 install black # pre-commit hook requires this
 ln -s -f ../../.pre-commit.sh .git/hooks/pre-commit
```

Comment:
do we want pip3 install autopep8 in the setup script?

Reply:
I'd rather have people install it globally so that if anyone ever updates requirements besides us we don't include `autopep8` (which is a dev dependency) in the requirements.txt

Reply:
sure, sounds reasonable